sdk/rust/oak/src/logger/mod.rs

```diff
@@ -87,6 +87,7 @@ pub fn init(level: Level) -> Result<(), SetLoggerError> {
     let (write_handle, read_handle) = crate::channel_create("Logger", &Label::public_untrusted())
         .expect("could not create channel");
     crate::node_create(
+        "log",
```

Comment:
Should it be called `logger`?

Reply:
Though it was called `log` already

Reply:
I don't have a strong preference, but I think keeping the original name is fine.